committee-steering/governance/annual-reports.md

```diff
@@ -62,45 +62,137 @@ of being more explicit than using our upstream shorthand or abbreviations.
-Examples?     
+## Questions for report:
+
+### Special Interest Groups:
+
+```
+# {sig-name} - {YYYY} annual report
+
+## Current initiatives
+
+1. What work did the SIG do this year that should be highlighted?
+
+   - 
+   - 
+   - 
+
+2. What initiatives are you working on that aren't being tracked in KEPs?
+
+   - 
+   - 
+   - 
+
+3. KEP work in {YYYY} (1.x, 1.y, 1.z): (generated)
+   - Stable
+      - [{name}]({KEP link}) - {release}
+      - [{name}]({KEP link}) - {release}
+   - Beta
+      - [{name}]({KEP link}) - {release}
+      - [{name}]({KEP link}) - {release}
+   - Alpha
+      - [{name}]({KEP link}) - {release}
+      - [{name}]({KEP link}) - {release}
+   - Pre-alpha
+      - [{name}]({KEP link})
+
+## Project health
+
+1. What areas and/or subprojects does your group need the most help with?
+   Any areas with 2 or fewer OWNERs? (link to more details)
+
+   - 
+   - 
+   - 
+
+2. What metrics/community health stats does your group care about and/or measure?
+
+   - 
+   - 
+   - 
+
+3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing 
+   to activities or programs that provide useful context or allow easy participation?
+
+   - 
+
+4. Does your [CONTRIBUTING.md] help **existing** contributors grow throughout the [contributor ladder]
+   by documenting any special training, requirements, or processes unique to reviewers/approvers in your group?
+
+   - 
+
+5. Does the group have contributors from multiple companies/affiliations?
+
+   - 
+
+6. Are there ways end users/companies can contribute that they currently are not?
+   If one of those ways is more full time support, what would they work on and why?
+
+   - 
+   - 
+
+## Membership
+
+- Primary slack channel member count: 
+- Primary mailing list member count: 
+- Primary meeting attendee count (estimated, if needed): 
+- Primary meeting participant count (estimated, if needed): 
+- Unique reviewers for SIG-owned packages: {generated}
+- Unique approvers for SIG-owned packages: {generated}
+
+Include any other ways you measure group membership
+
+## Subprojects (generated)
+
+New in {YYYY}:
+- {name}
+- {name}
+
+Retired in {YYYY}:
+- 
+- 
+
+Continuing:
+- 
+- 
+
+## Working groups (generated)
+
+New in {YYYY}:
+- [{name}]({link to community repo dir}) {link to YYYY annual report}
+- 
+
+Retired in {YYYY}:
+- 
+- 
+
+Continuing:
+- 
+- 
+
+## Operational
+
+Operational tasks in [sig-governance.md]:
+
+[ ] [README.md] reviewed for accuracy and updated if needed
+[ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+    (or created if missing and your contributor steps and experience are different or more
+    in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
+[ ] Subprojects in [sigs.yaml] reviewed for accuracy and updated if needed
+[ ] SIG leaders (chairs, tech leads, and subproject owners) in [sigs.yaml] are accurate and active, and updated if needed
```

Comment:
The list of OWNERS per sig is getting stale ( https://github.com/kubernetes/community/issues/6307 ), we should add a operational task for the leads to update/review this list as well. (in sigs.yaml)

Reply:
I think that's line 181, but will make it more explicit